frontend/packages/ceph-storage-plugin/locales/en/ceph-storage-plugin.json

```diff
@@ -518,7 +518,7 @@
   "Openshift Container Storage": "Openshift Container Storage",
   "Standard": "Standard",
   "Data will be ingested by Multi-cloud object gateway, deduped, compressed and encrypted. The encrypted chunks would be saved on the selected backing stores. Best used when the applications would always use the OpenShift Container Storage endpoints to access the data.": "Data will be ingested by Multi-cloud object gateway, deduped, compressed and encrypted. The encrypted chunks would be saved on the selected backing stores. Best used when the applications would always use the OpenShift Container Storage endpoints to access the data.",
-  "Data will be stored as is(no dedupe, compression, encryption) on the namespace stores. Namespace buckets allow for connecting to existing data and serving from them. Best used for existing data or when other applications (and native cloud services) need to access the data from outside the OpenShift Container Storage.": "Data will be stored as is(no dedupe, compression, encryption) on the namespace stores. Namespace buckets allow for connecting to existing data and serving from them. Best used for existing data or when other applications (and native cloud services) need to access the data from outside the OpenShift Container Storage.",
+  "Data will be stored on the NamespaceStores without performing de-duplication, compression, or encryption. BucketClasses of namespace type allow  connecting to existing data and serving from them. These are best used for existing data or when other applications (and cloud native services) need to access the data from outside the OpenShift Container Storage.": "Data will be stored on the NamespaceStores without performing de-duplication, compression, or encryption. BucketClasses of namespace type allow  connecting to existing data and serving from them. These are best used for existing data or when other applications (and cloud native services) need to access the data from outside the OpenShift Container Storage.",
```

Comment:
Data will be stored on the Namespace stores without performing de-duplication, compression, or encryption. Bucket classes of namespace type allow connecting to existing data and serving from them. These are best used for existing data or when other applications (and cloud-native services) need to access the data from outside the OpenShift Container Storage.